Frontend/src/screens/loginScreen.js

```diff
@@ -36,38 +43,117 @@ const LoginScreen = ({navigation}) => {
   };
 
   return (
-    <View>
-      <Text>Welcome back</Text>
-      <View>
-        <Text>Email</Text>
-        <TextInput
-          onChangeText={(UserEmail) => setUserEmail(UserEmail)}
-          placeholder="Enter Email"
-          returnKeyType="next"
-          blurOnSubmit={false}
-        />
-      </View>
-      <View>
-        <Text>Password</Text>
-        <TextInput
-          onChangeText={(UserPassword) => setUserPassword(UserPassword)}
-          placeholder="Enter Password"
-          blurOnSubmit={false}
-          returnKeyType="next"
-          secureTextEntry={true}
-        />
-      </View>
-      <Text onPress={() => navigation.navigate('')}>Forgot password?</Text>
-      <TouchableOpacity onPress={() => handleSubmitPress()}>
-        <Text>Sign in</Text>
-      </TouchableOpacity>
-      <View style={{flexDirection: 'row'}}>
-        <Text> Don't have an account? </Text>
-        <Text onPress={() => navigation.navigate('RegisterScreen')}>
-          Sign Up
-        </Text>
-      </View>
+    <View style={styles.container}>
+      <ImageBackground
+        source={require('../images/authBackground.jpg')}
+        style={styles.backgroundImage}>
+        <View style={styles.header}>
+          <Text style={styles.headerText}>Welcome back</Text>
+        </View>
+        <View style={styles.body}>
+          <Text style={styles.textBeforeTextInput}>Email</Text>
+          <TextInput
+            style={styles.textInput}
+            onChangeText={(UserEmail) => setUserEmail(UserEmail)}
+            placeholder="Enter Email"
+            placeholderTextColor="#6da7f2"
+            returnKeyType="next"
+            blurOnSubmit={false}
+          />
+          <Text style={styles.textBeforeTextInput}>Password</Text>
+          <TextInput
+            style={styles.textInput}
+            onChangeText={(UserPassword) => setUserPassword(UserPassword)}
+            placeholder="Enter Password"
+            placeholderTextColor="#6da7f2"
+            blurOnSubmit={false}
+            returnKeyType="next"
+            secureTextEntry={true}
+          />
```

Comment:
Can email and password be reused for register as well if you split only this part in another component?

Reply:
Yes, I can make a new component that contains only the email and password and reuse it for register as well.